cassiopeia/src/main/webapp/WEB-INF/spring/appServlet/servlet-context.xml

```diff
@@ -10,6 +10,9 @@
 		http://www.springframework.org/schema/util http://www.springframework.org/schema/util/spring-util.xsd
 		http://www.springframework.org/schema/security http://www.springframework.org/schema/security/spring-security-4.1.xsd
 		http://www.springframework.org/schema/aop http://www.springframework.org/schema/aop/spring-aop-3.2.xsd">
+		
+		<mvc:annotation-driven />
+		<context:component-scan base-package="edu.asu.diging.gilesecosystem.cassiopeia.web,edu.asu.diging.gilesecosystem.kafka.util.web" />
```

Comment:
I have changed the component scan in root context of cassiopeia to "edu.asu.diging.gilesecosystem.cassiopeia.core" and I included the controller component in the servelet-context.xml. I think this will prevent duplicacy. If I try to remove the controller component of cassiopeia from servelet-context, its returning 404 error. 
And regarding the annotation-diven tag, When you add <mvc:annotation-driven /> to your config, it replaces the default set of handler mappings and handler adapters, and those defaults were the ones that handled the old-style controllers. So when I remove it I am getting a 404 error.

Reply:
now the rest controller is missing which will break the system